meta-resin-common/recipes-core/systemd/systemd/coredump.conf

```diff
@@ -0,0 +1,2 @@
+[Coredump]
+Storage=none
```

Comment:
We could use journal to store the dump too which will rotate and avoid filling up the partition. What do you think?